cluster/common.sh

```diff
@@ -963,6 +969,31 @@ function create-certs {
   KUBECFG_KEY_BASE64=$(cat "${CERT_DIR}/pki/private/kubecfg.key" | base64 | tr -d '\r\n')
   KUBEAPISERVER_CERT_BASE64=$(cat "${CERT_DIR}/pki/issued/kube-apiserver.crt" | base64 | tr -d '\r\n')
   KUBEAPISERVER_KEY_BASE64=$(cat "${CERT_DIR}/pki/private/kube-apiserver.key" | base64 | tr -d '\r\n')
+
+  AGGR_CERT_DIR="${KUBE_TEMP}/easy-rsa-master/aggregator"
```

Comment:
Please add a little comment that we use another CA => another dir. Then it's fine.